x/cellarfees/module.go

```diff
@@ -84,24 +84,29 @@ type AppModule struct {
 	cdc           codec.Codec
 	accountKeeper types.AccountKeeper
 	bankKeeper    types.BankKeeper
+	corkKeeper    types.CorkKeeper
+	gravityKeeper types.GravityKeeper
 }
 
 // NewAppModule creates a new AppModule object
-func NewAppModule(keeper keeper.Keeper, cdc codec.Codec, accountKeeper types.AccountKeeper, bankKeeper types.BankKeeper) AppModule {
+func NewAppModule(keeper keeper.Keeper, cdc codec.Codec, accountKeeper types.AccountKeeper, bankKeeper types.BankKeeper,
+	corkKeeper types.CorkKeeper, gravityKeeper types.GravityKeeper) AppModule {
 	return AppModule{
 		AppModuleBasic: AppModuleBasic{},
 		keeper:         keeper,
 		cdc:            cdc,
 		accountKeeper:  accountKeeper,
 		bankKeeper:     bankKeeper,
+		corkKeeper:     corkKeeper,
+		gravityKeeper:  gravityKeeper,
```

Comment:
Similarly here, I imagine this needs a handle on the auction keeper.

Reply:
Bumping this as well -- `AppModule` needs the auction keeper, yes?

Reply:
weird, I recall adding this.